test/kms/bench_test.go

```diff
@@ -42,17 +42,6 @@ func BenchmarkKMS(b *testing.B) {
 	client, err := privval.NewSignerClient(endpoint, chainID)
 	require.NoError(b, err)
 
-	// ensure connection and warm up
-	b.Run("Ping", func(b *testing.B) {
```

Comment:
Is it possible to leave the Ping API benchmark using `endpoint` even if SignerClient.Ping() is removed?

Reply:
OK, I will implement `Ping` for benchmark.